DataSetExplorer/ConsoleUI.cs

```diff
@@ -0,0 +1,293 @@
+                            }
+                        }
+                        break;
+                    case "4":
+                        severityId = GetId("Severity");
+                        if (severityId.HasValue)
+                        {
+                            significanceResult = annotationConsistencyService.CheckMetricsSignificanceBetweenAnnotatorsForSeverity(severityId.Value);
+                            foreach (var result in significanceResult.Value)
+                            {
+                                Console.WriteLine(result.Key);
+                                result.Value.ToList().ForEach(pair => Console.WriteLine(pair.Key + "\n" + pair.Value));
+                            }
+                        }
+                        break;
+                    case "x":
+                        break;
+                    default:
+                        ChosenOptionErrorMessage(chosenOption);
+                        break;
+                }
+            } while (!chosenOption.Equals("x"));
+        }
+
+        private static int? GetId(string name)
+        {
+            string id = GetAnswerOnQuestion(name + " ID: ");
+            if (int.TryParse(id, out int parsedId)) return parsedId;
+            Console.Write("Invalid ID: ID must be an integer.");
+            return null;
+        }
+
+        private static string GetAnswerOnQuestion(string question)
+        {
+            Console.Write(question);
+            return Console.ReadLine();
+        }
+
+        private static void WriteMenuWithOptions()
+        {
+            Console.WriteLine("DATASET EXPLORER\n");
+
+            Console.WriteLine("Choose an option:");
+            Console.WriteLine("1. Create dataset");
+            Console.WriteLine("2. Analyze dataset");
+            Console.WriteLine("3. Export dataset");
+            Console.WriteLine("4. Check annotations consistency");
+            Console.WriteLine("x. Exit\n");
+        }
+
+        private static void WriteAnalyzeDataSetOptions()
+        {
+            Console.WriteLine("\nChoose an option:");
+            Console.WriteLine("1. Find instances requiring additional annotation");
+            Console.WriteLine("2. Find instances with all disagreeing annotations");
+            Console.WriteLine("x. Exit\n");
+        }
+
+        private static void WriteAnnotationsConsistencyOptions()
+        {
+            Console.WriteLine("\nChoose an option:");
+            Console.WriteLine("1. Check annotation consistency for a single annotator");
+            Console.WriteLine("2. Check annotation consistency between annotators");
+            Console.WriteLine("3. Check metrics significance for a single annotator");
+            Console.WriteLine("4. Check metrics significance between annotators");
+            Console.WriteLine("x. Exit\n");
+        }
+
+        private static void ChosenOptionErrorMessage(string chosenOption)
```

Comment:
This class is already quite large and there is room to grow. Does it make sense to extract a major submenu of this class into a separate class?